app/src/main/java/org/eyeseetea/malariacare/DashboardActivity.java

```diff
@@ -819,4 +815,43 @@ public void run() {
             }
         }, 1000);
     }
+
+    @Override
+    public void OnOrgUnitSelected(OrgUnit orgUnit) {
+        Log.d(TAG,"OnOrgUnitSelected");
+        initOrgUnitFragment(orgUnit);
+    }
+
+    public void initPlanned(){
+        //Show plannedFragment layout and hide plannedOrgUnitsFragment
+        LinearLayout changeLayout = (LinearLayout) findViewById(R.id.dashboard_planning_orgunit);
+        changeLayout.setVisibility(View.GONE);
+        changeLayout = (LinearLayout) findViewById(R.id.dashboard_planning_init);
+        changeLayout.setVisibility(View.VISIBLE);
+        plannedFragment = new PlannedFragment();
+        plannedFragment.setArguments(getIntent().getExtras());
+        replaceListFragment(R.id.dashboard_planning_init, plannedFragment);
+    }
+
+    private void initOrgUnitFragment(OrgUnit orgUnit) {
+        //hide plannedFragment layout and show plannedOrgUnitsFragment
+        LinearLayout changeLayout = (LinearLayout) findViewById(R.id.dashboard_planning_init);
```

Comment:
@idelcano What is going on here? Take a look at these lines. Too crazy to merge it :)